Added Accrualify section (#574)
mattermost/mattermost-handbook: operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md

> ## How to Review a Vendor Invoice

Reviewer: ```suggestion
## How to Review a vendor invoice
```

> If you need to review an invoice, Accrualify will send you an email alerting you that an invoice is on your queue for review and approval. The email will include the link “Review Invoice Request”, which will direct you to the invoice that needs your attention.

Reviewer: ```suggestion
If you need to review a vendor invoice, Accrualify will send you an email alerting you that an invoice is ready for review and approval. From the email, select the “Review Invoice Request” link to go directly to the invoice that needs your attention.
```

![](../../../../.gitbook/assets/How-to-review-a-vendor-invoice-1.PNG)

> Once you're directed to the pending invoice, please review the following details:

Reviewer: ```suggestion
Once you're directed to the pending vendor invoice, please review the following details:
```

> 1. **Vendor:** Please make sure you are reviewing an invoice for the vendor that you have submitted a PO for.

Reviewer: ```suggestion
1. **Vendor:** Make sure you're reviewing a vendor invoice that has a submitted purchase order.
```

> 2. **Amount:** Please make sure the amount on the invoice matches the PO submitted.

Reviewer: ```suggestion
2. **Amount:** Make sure the amount on the vendor invoice matches the submitted purchase order.
```

> 3. **Department:** Please make sure the department this spend is going to is your department or is properly allocated to your department if the spend is shared with other departments.

Reviewer: ```suggestion
3. **Department:** Make sure this spend is for your department (or is properly allocated to your department in cases where the spend is shared with other departments).
```

> 4. **Description:** Please make sure the description on the invoice is accurate (for example, it describes the service performed by the vendor for Mattermost).

Reviewer: ```suggestion
4. **Description:** Make sure the description on the invoice is accurate. For example, a description of the service performed by the vendor for Mattermost.
```

![](../../../../.gitbook/assets/How-to-review-a-vendor-invoice-2.PNG)

> Once you have reviewed the invoice and are ready to approve, click **Approve and Next**. After you approve the invoice, a Confirmation will pop up to ask **Are you sure you want to approve (invoice xxx)?**. Click **Ok**.

Reviewer: ```suggestion
Once you have reviewed the vendor invoice and are ready to approve it, select **Approve and Next**. 
When prompted to confirm the approval, select **OK**.
```

![](../../../../.gitbook/assets/How-to-review-a-vendor-invoice-3.PNG)

> If you need to reject the invoice, click **Reject and Next**. If you reject the invoice, it means you disagree with one or more items on the invoice. When you reject the invoice, please provide the reasons for the reject in the free text box that appears after you click on **Reject and Next**.

Reviewer: ```suggestion
If you need to reject the vendor invoice, select **Reject and Next**. Rejecting the invoice means you disagree with one or more invoice details. 

When prompted, please provide the reasons for rejecting the invoice in the **Rejection Comments** text field then select **OK**.
```

![](../../../../.gitbook/assets/How-to-review-a-vendor-invoice-4.PNG)

> To check for other open items on your dashboard, click on **Dashboard** on the left hand side of the screen. From the Dashboard, you will see other Actionable Items.

Reviewer: ```suggestion
, Select **Dashboard** to return to your dashboard and check for other open items that need your attention.
```
